src/providers/json-rpc-provider.ts

```diff
@@ -115,12 +115,20 @@ export class JsonRpcProvider extends Provider {
         return await this.sendJsonRpc('EXPERIMENTAL_genesis_config', []);
     }
 
+    /**
+     * Gets EXPERIMENTAL_light_client_proof from RPC
```

Comment:
Would be nice to provide a bit more info on what is this about (maybe link to relevant nearcore info?).